### PR TITLE
add profile for hattie

### PR DESCRIPTION
**Pull request by harrietrichardson**

### CODE REVIEW

File: content/authors/hattie-richardson/_index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+# Name
+title: Hattie Richardson
+# Name of the folder
+authors:
+- leah-walker
```
Comment (DominiqueMakowski) [inline content/authors/hattie-richardson/_index.md]:
```suggestion
- hattie-richardson
```

**Comment by DominiqueMakowski** [comment]
@harrietrichardson could you also edit the _index.md file?

**Comment by DominiqueMakowski** [comment]
That was good, don't worry about the workflow email it's irrelevant

**Comment by harrietrichardson** [comment]
Yay thank you!

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**